### ISSUE TITLE
Invalid .desktop file

### ISSUE THREAD

**Issue by pypingou** [Guake/guake#500]
While building 0.6.0 for Fedora I ran into:
````
desktop-file-install --vendor= --delete-original --dir=~/rpmbuild/BUILDROOT/guake-0.6.0-1.fc23.x86_64/usr/share/applications ~/rpmbuild/BUILDROOT/guake-0.6.0-1.fc23.x86_64/usr/share/applications/guake.desktop
~/rpmbuild/BUILDROOT/guake-0.6.0-1.fc23.x86_64/usr/share/applications/guake.desktop: error: file contains key "_Name" in group "Desktop Entry", but key names must contain only the characters A-Za-z0-9- (they may have a "[LOCALE]" postfix)
~/rpmbuild/BUILDROOT/guake-0.6.0-1.fc23.x86_64/usr/share/applications/guake.desktop: error: file contains key "_Comment" in group "Desktop Entry", but key names must contain only the characters A-Za-z0-9- (they may have a "[LOCALE]" postfix)
~/rpmbuild/BUILDROOT/guake-0.6.0-1.fc23.x86_64/usr/share/applications/guake.desktop: error: file contains key "_Keywords" in group "Desktop Entry", but key names must contain only the characters A-Za-z0-9- (they may have a "[LOCALE]" postfix)
~/rpmbuild/BUILDROOT/guake-0.6.0-1.fc23.x86_64/usr/share/applications/guake.desktop: error: required key "Name" in group "Desktop Entry" is not present
Error on file "~/rpmbuild/BUILDROOT/guake-0.6.0-1.fc23.x86_64/usr/share/applications/guake.desktop": Failed to validate the created desktop file
error: Bad exit status from /var/tmp/rpm-tmp.D5zGkS (%install)
````
So apparently https://github.com/Guake/guake/commit/efc129bc0d703041259e3236af02d9ddeafe0830 made the desktop file invalid

**Comment by pypingou** [comment]
Note: this is valid for both desktop files

**Comment by pypingou** [comment]
This also means we actually lost translations in the desktop file:

0.6.0:
````
[Desktop Entry]
Encoding=UTF-8
Name=Guake Terminal
Comment=Use the command line in a Quake-like terminal
TryExec=guake
Exec=guake
Icon=guake
Type=Application
Categories=GNOME;GTK;System;Utility;TerminalEmulator;
StartupNotify=true
Keywords=Terminal;Utility;
X-Desktop-File-Install-Version=0.22
````

Previously
````
[Desktop Entry]
Encoding=UTF-8
Name=Guake Terminal
Name[pt]=Guake Terminal
Name[pt_BR]=Guake Terminal
Name[fr]=Guake Terminal
Name[fr_FR]=Guake Terminal
Comment=Use the command line in a Quake-like terminal
Comment[pt]=Utilizar a linha de comando em um terminal estilo Quake
Comment[pt_BR]=Utilizar a linha de comando em um terminal estilo Quake
Comment[fr]=Utilisez la ligne de commande comme dans un terminal quake
Comment[fr_FR]=Utilisez la ligne de commande comme dans un terminal quake
TryExec=guake
Exec=guake
Icon=guake
Type=Application
Categories=GNOME;GTK;System;Utility;TerminalEmulator;
StartupNotify=true
X-Desktop-File-Install-Version=0.15
````

**Comment by piotrdrag** [comment]
I think there is some automake stuff missing. This is how a proper .desktop file looks like:

https://git.gnome.org/browse/gnome-terminal/tree/gnome-terminal.desktop.in.in

Some resources:

https://git.gnome.org/browse/gnome-terminal/tree/Makefile.am

https://www.redhat.com/archives/fedora-trans-list/2009-August/msg00040.html

**Comment by Stibbons** [comment]
closed in favor of #501

**Comment by stouf** [comment]
The Gnome .desktop files made invalid in https://github.com/Guake/guake/commit/efc129bc0d703041259e3236af02d9ddeafe0830 are still in the package currently on ArchLinux mirrors (https://www.archlinux.org/packages/community/x86_64/guake/).
I don't know who is responsible for the packaging for the multiple Linux distributions, but just wanted to let you guys know about the issue.
